src/components/SearchPage.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div id="search_page">
+    <custom-header />
+    <result-control :first_option="sort_by_realesed_date" :second_option="sort_by_rating" :options_name="sort_by" />
+    <film-gallery :cards="cards" />
+    <custom-footer :company_name="company_name" :app_name="app_name" />    
+  </div>
+</template>
+
+<script>
+import CustomHeader from './CustomHeader.vue';
+import CustomFooter from './CustomFooter.vue';
+import FilmGallery from './FilmGallery.vue';
+import ResultControl from './ResultControl.vue';
+import { EventBus } from '../event-bus';
+export default {
+  name: "SearchPage",
+  components: {
+    CustomHeader,
+    CustomFooter,
+    ResultControl,
+    FilmGallery
+  },
+  data() {
+    return {
+      sort_by_realesed_date: "REALESED DATE",
+      sort_by_rating: "RATING",
+      sort_by: "sort_by",
+      cards: [],
+      company_name: "netflix",
+      app_name: "roulette",
+    };
+  },
+  mounted() {
+    EventBus.$on("getMoviesByOption", (data) => {
+      if (data.input && data.input != "" && data.option) {
+        this.cards = [
+          {
```

Comment:
Under "src/core" folder that you create after one of the above comments consider creating a folder "data" that will hold any number of file to your liking - as long as they contain all the hard-coded data that you use throughout the app. The idea is to create a central place for the mock data to be reused in components, tests, etc instead of real data.